tendrl/commons/objects/__init__.py

```diff
@@ -201,7 +202,9 @@ def load(self):
                 pass
 
         _copy = self._copy_vars()
-
+        # Check if self.value already set, use it
+        if self.value.find('{') < 0:
```

Comment:
`_copy_vars()` sets the old `value` field again in `_copy`. So I check if self.value doesn't contain `{` it means its properly rendered and so I assign it back to `_copy` variable's `value` field.